scripts/test

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Turn on strict mode.
+set -eEo pipefail
+
+# Run the tests.
+istanbul cover jasmine-node node_modules/jasmine-test-helpers/lib/ --captureExceptions spec/
+
+# Check the code coverage.
+if [[ "$CI" == "true" ]]; then
+    codecov
+else
+    # Easiest way to generate coverage files and not upload the report
+    # is to use --dump.
+    echo "Executing codecov and generating coverage report."
+    codecov --dump > /dev/null
+fi
```

Comment:
I've seen this implemented differently in Opentoken and something I reviewed for Kyle yesterday. Here's how it was done in those places: 

```
 if [[ "$CI" == "true" ]]; then
     codecov
 elif [[ -d ".git" ]]; then
     # Easiest way to generate coverage files and not upload the report
     # is to use --dump.
     echo "Executing codecov and generating coverage report."
     codecov --dump > /dev/null
 else
     # codecov runs git and hg(mercurial).  If the folder is not under source
     # control, codecov breaks with some nasty error.  This situation happens
     # during deployment, so we just don't worry about coverage during
     # deployment.
     echo "Not running code coverage report."
 fi
```